src/parquet/column_reader.cc

```diff
@@ -51,6 +51,12 @@ ColumnReader::ColumnReader(const parquet::ColumnMetaData* metadata,
     case CompressionCodec::SNAPPY:
       decompressor_.reset(new SnappyCodec());
       break;
+    case CompressionCodec::LZO:
+      decompressor_.reset(new Lz4Codec());
+      break;
```

Comment:
The decompressor was already available, but never initialized here. I'm going to rebase this patch after #38 and add unit tests for the decompression code path using a test fixture (unless someone else wants to do that)